functions/src/index.ts

```diff
@@ -98,7 +99,34 @@ userApp.get('/users/:userId/invites', (request, response) => {
 })
 
 userApp.get('/users/:userId/invites/:invitationId', (request, response) => {
-  response.send("hello world /users/:userId/invites/:invitationId")
+  firestore
+    .collection('users').doc(request.params.userId)
+    .collection('invites').doc(request.params.invitationId).get().then(invitationDoc => {
+      //This should actually be rendered by React SSR
+      const html =
+        '<!DOCTYPE html>' +
+        '<html>' +
+        '  <head>' +
+        '    <meta charset="UTF-8">' +
+        '    <meta name="twitter:card" content="' + invitationDoc.data()['twitter:card'] + '" />' +
+        '    <meta name="twitter:site" content="' + invitationDoc.data()['twitter:site'] + '" />' +
+        '    <meta name="twitter:creator" content="' + invitationDoc.data()['twitter:creator'] + '" />' +
+        '    <meta property="og:url" content="' + invitationDoc.data()['og:url'] + '" />' +
+        '    <meta property="og:title" content="' + invitationDoc.data()['og:title'] + '" />' +
+        '    <meta property="og:description" content="' + invitationDoc.data()['og:description'] + '" />' +
+        '    <meta property="og:image" content="' + invitationDoc.data()['og:image'] + '" />' +
```

Comment:
These actually populate `unknown` meta tag values if they are missing. Better to give HTTP 500 rather than returning a page with unknown meta tags??


![image](https://user-images.githubusercontent.com/7414320/48740251-19657b80-ec9a-11e8-8b2b-162d2bc21eea.png)